templates/search_results.hbs

```diff
@@ -5,103 +5,125 @@
+            {{/is}}
+          </ul>
+        </section>
+      {{/if}}
+    </section>
+
     <section class="search-results-column">
-      <h2 class="search-results-subheading">
-        {{t 'knowledge_base'}}
-      </h2>
-      {{#if article_results}}
+      <h1 class="search-results-subheading">
+        {{#is current_filter.identifier 'unified'}}
+          {{t 'results' query=query count=results_count}}
+        {{else}}
+          {{#unless current_subfilter.identifier}}
+            {{t 'results' query=query count=results_count}}
+          {{else}}
+            {{t 'results_with_scope' query=query count=results_count scope_name=current_subfilter.name}}
+          {{/unless}}
+        {{/is}}
+      </h1>
+      {{#if results}}
         <ul class="search-results-list">
-          {{#each article_results}}
-            <li class="search-result">
-              <a href="{{url}}" class="search-result-link">{{title}}</a>
-              {{#if vote_sum}}
-                <span class="search-result-votes meta-count">{{vote_sum}}</span>
-              {{/if}}
-              <ul class="meta-group">
-                <li>
-                  <ol class="breadcrumbs search-result-breadcrumbs">
-                    {{#each path_steps}}
-                      {{#is url '#'}}
-                        <li title="{{name}}">{{name}}</li>
-                      {{else}}
+          {{#each results}}
+            <li class="search-result-list-item result-{{type}}">
+              <h2 class="search-result-title">
+                <a href="{{url}}" class="results-list-item-link">{{title}}</a>
+              </h2>
+              <div class="search-result-icons">
+                {{#if vote_sum}}
+                  <span class="search-result-votes">{{vote_sum}}</span>
+                {{/if}}
+                {{#if comment_count}}
+                  <span class="search-result-meta-count">
+                    {{comment_count}}
+                  </span>
+                {{/if}}
+              </div>
+              <article>
+                <ul class="meta-group">
+                  <li>
+                    <ol class="breadcrumbs search-result-breadcrumbs">
+                      {{#each path_steps}}
                         <li title="{{name}}"><a href="{{url}}">{{name}}</a></li>
-                      {{/is}}
-                    {{/each}}
-                  </ol>
-                </li>
-                <li class="meta-data">
-                  {{#link 'user_profile' id=author.id}}
-                    {{author.name}}
-                  {{/link}}
-                </li>
-                <li class="meta-data">{{date created_at timeago=true}}</li>
-              </ul>
-              <div class="search-result-description">{{text}}</div>
+                      {{/each}}
+                    </ol>
+                  </li>
+                  <li class="meta-data">{{author.name}}</li>
```

Comment:
When using a screen reader, this sudden name doesn't make sense. Can be solved with an aria-label="authored by"